### PR TITLE
chore(build-and-release-stable): ignore rc releases

### PR DESCRIPTION
**Pull request by chenrui333**
**Notes for Reviewers**

- Update stable release flow to ignore rc releases

relates to https://github.com/Homebrew/homebrew-core/pull/157503

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .github/workflows/build-and-release-stable.yml [hunk 1]
```diff
@@ -140,49 +140,48 @@ jobs:
           version: latest
           args: release --clean --skip=validate
       - name: bump homebrew-core formula
-        uses: mislav/bump-homebrew-formula-action@v2
+        uses: mislav/bump-homebrew-formula-action@v3
         with:
           formula-name: mesheryctl
           download-url: https://github.com/meshery/meshery.git
-          formula-path: Formula/m/mesheryctl.rb
```
Comment (chenrui333) [inline .github/workflows/build-and-release-stable.yml:147]: this got fixed per https://github.com/mislav/bump-homebrew-formula-action/pull/59 in 2.3 release

File: .github/workflows/build-and-release-stable.yml [hunk 2]
```diff
@@ -140,49 +140,48 @@ jobs:
           version: latest
           args: release --clean --skip=validate
       - name: bump homebrew-core formula
-        uses: mislav/bump-homebrew-formula-action@v2
+        uses: mislav/bump-homebrew-formula-action@v3
```
Comment (chenrui333) [inline .github/workflows/build-and-release-stable.yml:143]: also bump it to use nodejs 20 runtime with v3

**Comment by hamza-mohd** [comment]
Thank you!

File: .github/workflows/build-and-release-stable.yml (hunk 2 above)
Comment (leecalcote) [reply .github/workflows/build-and-release-stable.yml:143]: Good call. 👍

**Comment by leecalcote** [comment]
Thank you for taking care of this, @chenrui333. 👍

**Pull request merged**